python/lsst/ts/IntegrationTests/base_script.py

```diff
@@ -25,6 +25,7 @@
 from lsst.ts.idl.enums.Script import ScriptState
 
 from datetime import date
+import copy
```

Comment:
We should have `isort` setup for this repo.

Reply:
I have a task for next sprint to implement pre_commit and stuff on this package.